traffic_ops/testing/api_contract/v4/test_cdns.py

```diff
@@ -98,17 +64,25 @@ def test_cdn_contract(
 		cdn_keys = set(first_cdn.keys())
 
 		logger.info("CDN Keys from cdns endpoint response %s", cdn_keys)
+		response_template = response_template_data.get("cdns").get("properties")
 		# validate cdn values from prereq data in cdns get response.
 		prereq_values = [
 			cdn_post_data["name"],
 			cdn_post_data["domainName"],
 			cdn_post_data["dnssecEnabled"]
 		]
 		get_values = [first_cdn["name"], first_cdn["domainName"], first_cdn["dnssecEnabled"]]
-		# validate data types for values from cdn get json response.
-		for (prereq_value, get_value) in zip(prereq_values, get_values):
-			assert isinstance(prereq_value, type(get_value))
-		assert cdn_keys == set(cdn_post_data.keys())
+		get_types = {}
+		for key in first_cdn:
+			get_types[key] = first_cdn[key].__class__.__name__
```

Comment:
Here , I'm using magic methods to extract value's datatype.
if we use type(value) it will return eg: <class str>, But the requirement is to extract the type name, So i've used magic methods to extract class name from value.
eg: 
data = "bla bla"
print(type(data))
print(data.__class__.__name__)
output:
<class 'str'>
str

Reply:
data = "bla bla"
print(type(data).__name__)
print(data.__class__.__name__)

Reply:
Sure, Will change it to
`type(data).__name__`